requirements.txt

```diff
@@ -1,2 +1,2 @@
--e git+https://github.com/CartoDB/carto-python.git#egg=carto
+-e git+https://github.com/CartoDB/carto-python.git@service-account-endpoint#egg=carto
```

Comment:
After merging, looks like we have to update the dependency.

Reply:
Yes, we have